bundles/org.openhab.binding.samsungtv/README.md

```diff
@@ -26,81 +31,639 @@ Tested TV models:
 | UE58RU7179UXZG | PARTIAL | Supported channels: `volume`, `mute`, `power`, `keyCode` (at least)                                                                                    |
 | UN50J5200      | PARTIAL | Status is retrieved (confirmed `power`, `media title`). Operating device seems not working.                                                            |
+| UN46EH5300     | OK      | All channels except `programTitle` and `channelName` are working                                                                                       |
+| UE75MU6179     | PARTIAL | All channels except `brightness`, `contrast`, `colorTemperature` and `sharpness`                                                                       |
+| QN55LS03AAFXZC | PARTIAL | Supported channels: `volume`, `mute`, `keyCode`, `power`, `artMode`, `url`, `artImage`, `artLabel`, `artJson`, `artBrightness`,`artColorTemperature`   |
+| QN43LS03BAFXZC | PARTIAL | Supported channels: `volume`, `mute`, `keyCode`, `power`, `artMode`, `url`                                                                             |
+
+If you enable the Smartthings interface, this adds back the `sourceName`, `sourceId`, `programTitle` and `channelName` channels on >2016 TV's
+Samsung removed the app API support in >2019 TV's, if your TV is >2019, see the section on [Apps](#apps).
+Samsung removed the art API support in >2021 TV's, if your TV is >2021, see the section on [setArtMode](#setArtMode) .
+
+**NOTE:** `brightness`, `contrast`, `colorTemperature` and `sharpness` channels only work on legacy interface TV's (<2016).
 
 ## Discovery
 
-The TV's are discovered through UPnP protocol in the local network and all devices are put in the Inbox.
+The TV's are discovered through UPnP protocol in the local network and all devices are put in the Inbox. TV must be ON for this to work.
 
 ## Binding Configuration
 
-The binding does not require any special configuration.
+Basic operation does not require any special configuration.  
+For >2019 TV's, the app workaround uses a built in list of known app ID's. You can add to this list by editing the `misc/samsungtv.applist` file in the openhab config directory.  
+To enable Smartthings polling, you have to add a Smartthings Personal Access token (PAT) to the Thing configuration.  
 
 ## Thing Configuration
 
-The Samsung TV Thing requires the host name and port address as a configuration value in order for the binding to know how to access it.
-Samsung TV publish several UPnP devices and hostname is used to recognize those UPnP devices.
-Port address is used for remote control emulation protocol.
-Additionally, a refresh interval can be configured in milliseconds to specify how often TV resources are polled.
+The Samsung TV Thing requires the host name and port address as a configuration value in order for the binding to know how to access it.  
```

Comment:
```suggestion
The Samsung TV Thing requires the host name and port address as a configuration value in order for the binding to know how to access it.
```

Reply:
Fixed.

Reply:
I would note that these spaces are used to separate paragraphs though, so this becomes one long paragraph.